addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/protocol/MilightDiscover.java

```diff
@@ -75,7 +75,7 @@
     public MilightDiscover(DiscoverResult discoverResult, int resendTimeoutInMillis, int resendAttempts)
             throws SocketException {
         this.resendAttempts = resendAttempts;
-        this.resendTimeoutInMillis = Math.min(resendTimeoutInMillis, 200);
```

Comment:
I thing I wanted to use a `Math.max` here, so that a user cannot set a value lower than 200ms. Nice catch that something is wrong here though.

Reply:
The value passed into the constructor looks hard coded to me in the two places it's used (MilightBridgeV3Handler and MilightBridgeDiscovery), so as the code is now it doesn't seem possible for a user to be able to set it.

Reply:
I haven't looked into the codebase for some time now. I was sure it was configurable, if it is not, it should be. But the Math is not necessary then of course.

Reply:
The only one I can find is indeed, 

https://github.com/openhab/openhab2-addons/blob/0647af745a131d0094e042180cb387318fa2cf3a/addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/handler/MilightBridgeV3Handler.java#L93